examples/tvm/mnist/tvm_mnist.py

```diff
@@ -0,0 +1,167 @@
+import sys
+import numpy as np
+
+
+import pickle 
+
+import json
+
+from mnist import MNIST
+import pathlib
+
+from tvm import relay
+from tvm.relay import testing
+import tvm
+from tvm import te
+import tvm.contrib.graph_executor as runtime
+
+def loadMnist(path, dataset='test'):
```

Comment:
Why are there two copies of this function (there's also one in parser_utils.py)

Reply:
Both of them need loadMnist, hence both of them have it. It's possible to have tvm_mnist.py use the one in parserUtils, but I kept them separate. Do you think I should change this?

Reply:
it's fine for now I think